pipeline/metadata/schema.py

```diff
@@ -0,0 +1,299 @@
+"""Schema for various data types passed around in the pipeline and bigquery"""
+
+from __future__ import absolute_import
+from __future__ import annotations  # required to use class as a type inside the class
+
+import dataclasses
+from dataclasses import dataclass
+from typing import Optional, List, Dict, Any
+
+# pylint: disable=too-many-instance-attributes
+
+
+@dataclass
+class SatelliteAnswer():
+  """Class for keeping track of Satellite answer content"""
+  # Keys
+  ip: str
+  # Metadata
+  asnum: Optional[int] = None
+  asname: Optional[str] = None
+  http: Optional[str] = None
+  cert: Optional[str] = None
+  matches_control: Optional[str] = None
+
+  def add_tag_to_answer(self, tag: SatelliteAnswerMetadata) -> None:
```

Comment:
Make this a standalone method. No need to be part of the class. It makes the decoupling clearer.

Reply:
Should it be `_tags_`?

Reply:
made standalone, and changed name to `tags`